Baikal/PostEffects/ML/denoiser.cpp

```diff
@@ -213,6 +223,43 @@ namespace Baikal
             }
         }
 
+
+        void MLDenoiser::WriteToTensor(CLWBuffer<RadeonRays::float3> src_buffer,
```

Comment:
Let's call it `WriteToInputs`, and rename the `m_sample_count_cache` like `m_inputs_cache`.

Reply:
Done